### PR TITLE
Update ML Dev Exp SIG README.md

### PR DESCRIPTION
**Pull request by andrewballantyne**
Initial updates to correct for out of date information.

### CODE REVIEW

**Review by kywalker-rh — APPROVED** [review]
lgtm

File: sig-ml-developer-experience/README.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 
 ### Chairs
+
 The Chairs of the SIG run operations and processes governing the SIG.
-- Jay Koehler
 - Kyle Walker
-- Erwan Granger
-- Romeo Kienzler
+- Andrew Ballantyne
+-
```
Comment (andrewballantyne) [inline sig-ml-developer-experience/README.md:19]: Removing Jay & Erwan from chairs as per their request.

Romeo has been inactive.

Comment (andrewballantyne) [reply sig-ml-developer-experience/README.md:19]: Also added myself as I have been participating for some time. This was done on a call with Kyle.

**Review by lucferbux — APPROVED** [review]
/lgtm

**Review by LaVLaS — APPROVED** [review]

**Pull request merged**